01_P/P_2_0_03/sketch.js

```diff
@@ -18,26 +18,27 @@
 
 /**
  * drawing with a changing shape by draging the mouse.
- *   
+ *
  * MOUSE
  * position x          : length
  * position y          : thickness and number of lines
  * drag                : draw
- * 
+ *
  * KEYS
  * 1-3                 : stroke color
  * del, backspace      : erase
  * s                   : save png
  */
+'use strict';
 
 var strokeColor = [];
 
 function setup() {
   createCanvas(720, 720);
   colorMode(HSB,360,100,100,100);
-  smooth();
```

Comment:
Removed smooth() since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).